lib/ansible/utils/vars.py

```diff
@@ -195,3 +200,28 @@ def isidentifier(ident):
         return False
 
     return True
+
+
+def validate_variable_names(names):
```

Comment:
need to x2 check, but i thought we were moving away from utils/ in favor of specifics for each subsystem (lib/ansible/vars)?

Reply:
Ok, I wasn't aware, I'll move it into `lib/ansible/vars/`.